### PR TITLE
refactoring get_individual (includes AI Export fix)

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
## Refactoring get_individual

- [x] agenda/agenda_items/(?P<pk>[0-9]+)/$
- [x] agenda/agenda_items/export/ (updated and fixed)
- [x] cdo/client/(?P<client_id>[0-9]+)/$
- [x] available_positions/(?P<pk>[0-9]+)/similar/
- [x] available_positions/archived/(?P<pk>[0-9]+)/ (updated, but still broken - but at least breaks match)
- [x] bureau/positions/(?P<pk>[0-9]+)/ ~(i think also broken, but again, consistent)~ not broken, thanks Scott!
- [x] positions/(?P<pk>[0-9]+)/
- [x] projected_vacancies/(?P<pk>[0-9]+)/

jumped between dev and this branch to confirm urls and their payloads were identical

# Current Status: Complete

### CODE REVIEW

File: talentmap_api/fsbid/services/available_positions.py [hunk 1]
```diff
@@ -365,7 +364,9 @@ def convert_ap_query(query, allowed_status_codes=["HS", "OP"], isTandem=False):
     '''
 
     prefix = ""
-
+    print('🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸')
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/available_positions.py:367]: 🐸

File: talentmap_api/fsbid/services/available_positions.py [hunk 2]
```diff
@@ -446,6 +447,9 @@ def convert_all_query(query):
     to convert_ap_query request_params.cps_codes of anything
     but FP, OP, or HS will get removed from query
     '''
+    print('👾👾👾👾👾👾👾👾👾👾👾')
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/available_positions.py:450]: 🎮

**Comment by elizabeth-jimenez** [comment]
Confirmed same payload after change

- [ ] available_positions/pk/similar

- [ ]

File: talentmap_api/fsbid/views/agenda.py [hunk 3]
```diff
@@ -18,13 +18,6 @@
 class AgendaItemView(BaseView):
     permission_classes = [Or(isDjangoGroupMember('cdo'), isDjangoGroupMember('ao_user'),)]
 
-    @swagger_auto_schema(
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/views/agenda.py:21]: removed cause single

**Comment by scott062** [comment]
This is all looking good, but I'm not seeing the same stuff "broken." We can ss and verify testing results

**Pull request merged**